Linked list Trickiest Questions/Detecting and Removing loop from ll.cpp

```diff
@@ -1,78 +1,98 @@
 #include<iostream>
 #include<conio.h>
 using namespace std;
+
 struct node
 {
-    int info;
-    node *next;
+  int info;
+  node *next;
 };
-node * create(int x)
+node *create (int x) //creating a new node
 {
-    node *p =new node;
-    p->info=x;
-    p->next=NULL;
-    return p;
+  node *p = new node;
+  p->info = x;
+  p->next = NULL;
+  return p;
 
 }
-void print (node *ptr)
+
+void print (node * ptr) //printing the linked list
 {
-    while(ptr!=NULL)
+  while (ptr != NULL)
     {

```

Comment:
Fix indentation of while statement.
It should be:
while
{

Reply:
Also, have the comments about the function, above its definition for better readability. Try following this with other general comments too.